source/mathematics/interactive.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Interactive sessions"
+---
+
+These are instructions on getting quick interactive sessions on BluePebble, assuming you already have login to this cluster and a project code. As this is primarily aimed for the School of Mathematics, we'll refer to the project code to be in the form MATHXXXXXX, but this could be in other format.
+
+The School of Mathematics ringfenced a few of its nodes with the software Magma installed on them into the "magma" partition, and we also use this partition for quick interactive testing.
+
+:::{important}
+If you need to run something heavy that takes hours or days please remove any reference to the "magma" partition from the below.
+:::
+
+We use maple/xmaple as an example but comsol, magma, math/mathematica, etc work similarly.
+
+- **If you don't need a graphical interface:**
+1. Connect to the login node, as described [here](https://www.acrc.bris.ac.uk/protected/hpc-docs/connecting/index.html). For this your don't need X11 forwarding.
+2. Once you are on the login node, do
+```console
+$ srun --account=MATHXXXXXX --partition=magma --mem=1gb --time=00:01:00 --pty bash -i
+```
+to get an interactive session on one of our ringfenced nodes for quick interactive sessions. (No heavy jobs are allowed to be run on the login node, hence this command.) Notice that the above gives you a terminal for 1 minute, obviously change this to what time you need and save your work often as your terminal will be killed after the requested time.
```

Comment:
Suggest some changes to wording to clarify the intended use of login nodes and the resources the command requests 

```suggestion
to get an interactive session on one of our ringfenced nodes for quick interactive sessions. Login nodes are intended for small scale interactive work. This command allows us to create an interactive session on a compute node for more intensive interactive work. Notice that the above gives you a terminal for 1 minute with 1 GB memory and the default number of CPU cores (1) -- please change this to specify the time and resources you need. Also, remember to save your work often as your terminal will be killed after the requested time.
```